src/chia_log/handlers/daily_stats/stats_manager.py

```diff
@@ -97,6 +110,9 @@ def _run_loop(self):
             if datetime.now() > self._datetime_next_summary:
                 self._send_daily_notification()
                 self._datetime_next_summary += timedelta(hours=self._frequency_hours)
+            if self._startup and datetime.now() > self._datetime_startup_summary:
+                self._send_daily_notification(startup=True)
+                self._startup = False
```

Comment:
See no need for an if-statement here, the flag can be reset directly in `_send_daily_notification`